src/application/App.tsx

```diff
@@ -108,9 +110,15 @@ function LeftSidePanelWrapper() {
     if (e.metaKey) {
       switch (e.key.toLowerCase()) {
         case '1':
-          return handleSideBarClick('Explorer');
+          handleSideBarClick('Explorer');
+          e.preventDefault();
+          e.stopPropagation();
+          break;
         case '2':
-          return handleSideBarClick('References');
+          handleSideBarClick('References');
+          e.preventDefault();
+          e.stopPropagation();
+          break;
```

Comment:
nit: you could prevent default and stop propagation before the switch case so you don't have to write it multiple times

Reply:
No, I don't want to do that for other shortcuts than CMD+1 and CMD+2.